frontend/public/components/filter-toolbar.tsx

```diff
@@ -100,8 +101,10 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
   const [inputText, setInputText] = React.useState('');
   const [filterType, setFilterType] = React.useState(FilterType.NAME);
   const [isOpen, setOpen] = React.useState(false);
-  const placeholder =
-    filterPlaceholder || (FilterType.NAME === filterType ? 'Search by name...' : 'app=frontend');
+  const [placeholder, setPlaceholder] = React.useState(
+    nameFilterPlaceholder ||
+      (FilterType.NAME === filterType ? 'Search by name...' : 'app=frontend'),
+  );
```

Comment:
Filter type is initialized to name, so you don't need to check it here.

```suggestion
  const [placeholder, setPlaceholder] = React.useState(nameFilterPlaceholder || 'Search by name...');
```

Reply:
done